packages/xstate-test/test/index.test.ts

```diff
@@ -202,7 +202,7 @@ it('prevents infinite recursion based on a provided limit', () => {
 });
 
 // TODO: have this as an opt-in
-it('executes actions', async () => {
+it.skip('executes actions', async () => {
```

Comment:
what changed here and why?

Reply:
See your comment:

```ts
// packages/xstate-test/src/machine.ts

export function executeAction(
  actionObject: BaseActionObject,
  state: AnyState
): void {
  // TODO: this is a hack, it doesn't correctly resolve actions
  // what gets passed as `action` here is probably also not correct
  if (typeof (actionObject as any)._exec === 'function') {
    (actionObject as any)._exec(state.context, state.event, {
      _event: state._event,
      action: actionObject,
      state
    });
  }
}
```

Reply:
We need to determine if executing actions is actually valuable in a testing scenario; I don't think it is. Tests verify; the only execution they should be responsible for is in the execution of external events.

Reply:
> Tests verify; the only execution they should be responsible for is in the execution of external events.

Makes sense, although then we should somehow just disallow actions in those test machine configs.

But in such a case, we should just remove this test and the mentioned hacky code. WDYT?

Reply:
Yes, let's remove it.

Reply:
@Andarist I will remove the test, but the executing action changes need to be made upstream on the `main` branch.

Reply:
For reference, this is removed here: https://github.com/statelyai/xstate/pull/3735